odl/operator/__init__.py

```diff
@@ -34,3 +34,5 @@
 __all__ += pspace_ops.__all__
 
 from . import oputils
```

Comment:
No, that's different. What I mean is that the *relative* import `from .local_pkg import stuff` also imports `local_pkg` into the namespace. That's not true for absolute imports.

Basically we can remove all of the `from . import subpackage` things.